stringtemplate4/src/main/java/org/jdbi/v3/stringtemplate4/StringTemplateSqlLocator.java

```diff
@@ -128,7 +125,8 @@ public static STGroup findStringTemplateGroup(String path) {
      * @return the loaded StringTemplateGroup.
      */
     public static STGroup findStringTemplateGroup(ClassLoader classLoader, String path) {
-        return CACHE.computeIfAbsent(path, p -> ThreadLocal.withInitial(() -> readStringTemplateGroup(classLoader, path))).get();
```

Comment:
and I think this one was a bug, different classloaders with the same path would both have the same map key before!